helpers/networker.py

```diff
@@ -57,15 +57,26 @@ def change_gro(self, dev, on=True):
     def change_gso(self, dev, on=True):
         self._set_state(dev, "gso", on)
 
-    def run_test_tso_gro_gso(self, client, server, test, mtu, tso, gro, gso):
+    def get_tcp_option(self, option_name):
+        cmd = f"sudo sysctl {option_name}"
```

Comment:
Please don't use `sudo`. Now we set `sudo` for all command in `run_cmd`

Reply:
Yes fixed